stable/mission-control/values.yaml

```diff
@@ -33,6 +33,37 @@ rbac:
       - watch
       - list
 
+networkpolicy:
+  # Allows all ingress and egress
+  - name: mission-control
+    podSelector:
+      matchLabels:
+        app: mission-control
+    egress:
+      - {}
+    ingress:
+      - {}
+  # Uncomment to allow only mission-control pods to communicate with postgresql (if postgresql.enabled is true)
+  # - name: postgres
+  #   podSelector:
+  #     matchLabels:
+  #       app: postgresql
+  #   ingress:
+  #   - from:
+  #     - podSelector:
+  #         matchLabels:
+  #           app: mission-control
+  # Uncomment to allow only mission-control pods to communicate with elasticsearch (if elasticsearch.enabled is true)
```

Comment:
@danielezer - This is not needed as elasticsearch service is deployed as part of mission-control pod.

Reply:
Thanks @amithins. Removed